knowledge/geography/attribution.txt

```diff
@@ -0,0 +1,5 @@
+Title of work: Brno-Bystrc
+Link to work: https://cs.wikipedia.org/wiki/Brno-Bystrc
```

Comment:
I'm sorry but we can not accept non-english references.

changing this to `en` changes the page, and doesn't have the data you have below.